### PR TITLE
fix(kuma-cp) fix env

### PR DESCRIPTION
**Pull request by lobkovilya**
Signed-off-by: Ilya Lobkov <ilya.lobkov@konghq.com>

### Summary

Fix env

### CODE REVIEW

**Review by nickolaev — COMMENTED** [review]
Do we want to expand testing and ensure that `loader_test.go` has all the env variables checked?

**Comment by lobkovilya** [comment]
Yes, I will expand testing

**Review by jakubdyszkiewicz — APPROVED** [review]

File: pkg/plugins/runtime/k8s/webhooks/injector/injector.go [hunk 1]
```diff
@@ -405,18 +410,13 @@ func (i *KumaInjector) NewAnnotations(pod *kube_core.Pod, mesh *mesh_core.MeshRe
 		annotations[metadata.CNCFNetworkAnnotation] = metadata.KumaCNI
 	}
 
-	// By default always set 'kuma.io/virtual-probes: enabled' if it's not set yet.
-	if _, exist, _ := metadata.Annotations(pod.Annotations).GetEnabled(metadata.KumaVirtualProbesAnnotation); !exist {
-		annotations[metadata.KumaVirtualProbesAnnotation] = metadata.AnnotationEnabled
+	if err := virtualProbesEnabled(annotations, pod, i.cfg); err != nil {
+		return nil, err
```
Comment (jakubdyszkiewicz) [inline pkg/plugins/runtime/k8s/webhooks/injector/injector.go:414]: please change the method names. I was confused why it's not returning bool and what is going on. It should be something like `setVirtualProbesEnabledAnnotation`, `setVirtualProbesPortAnnotation`

File: pkg/plugins/runtime/k8s/webhooks/injector/injector.go [hunk 2]
```diff
@@ -405,18 +410,13 @@ func (i *KumaInjector) NewAnnotations(pod *kube_core.Pod, mesh *mesh_core.MeshRe
...
 	}
-	// Disable virtual probes if pod has 'kuma.io/gateway' annotation, because we don't intercept inbound traffic
-	if enabled, exist, _ := metadata.Annotations(pod.Annotations).GetEnabled(metadata.KumaGatewayAnnotation); exist && enabled {
-		annotations[metadata.KumaVirtualProbesAnnotation] = metadata.AnnotationDisabled
+	if err := virtualProbesPort(annotations, pod, i.cfg); err != nil {
+		return nil, err
```
Comment (jakubdyszkiewicz) [inline pkg/plugins/runtime/k8s/webhooks/injector/injector.go:417]: give more context to these errors

File: test/testenvconfig/envconfig.go [hunk 3]
```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2013 Kelsey Hightower
```
Comment (jakubdyszkiewicz) [inline test/testenvconfig/envconfig.go:1]: Is this copy paste? Can you also paste the reference here (link to github)?

**Review by nickolaev — APPROVED** [review]

**Pull request merged**